.env

```diff
@@ -96,3 +96,5 @@ BLACKFIRE_SERVER_TOKEN=<your_backfire_server_token>
 
 WEBHOOK_BASIC_AUTH_USERNAME=webhook
 WEBHOOK_BASIC_AUTH_PASSWORD=cCVMxdX8hfctDefk
+
+AUTOMATE_USER_LIST_PATH=var/user-files/
```

Comment:
Maybe rename it to `GENERATED_USERS_FILE_PATH`, what do you think?

`automate` does mean much.